libursa/Responses.h

```diff
@@ -42,6 +42,7 @@ class Response {
                                          uint64_t iterator_position,
                                          uint64_t total_files);
     static Response ok();
+    static Response config(std::map<std::string, uint64_t> values);
```

Comment:
```suggestion
    static Response config(std::unordered_map<std::string, uint64_t> values);
```

This change requires `#include <unordered_map>`!